pkg/apis/workload/v1alpha1/workloadcluster_types.go

```diff
@@ -88,6 +88,19 @@ type WorkloadClusterStatus struct {
 	// A timestamp indicating when the syncer last reported status.
 	// +optional
 	LastSyncerHeartbeatTime *metav1.Time `json:"lastSyncerHeartbeatTime,omitempty"`
+
+	// SyncerVirtualWorkspaceURLs contains all syncer virtual workspace URLs.
+	// +optional
+	SyncerVirtualWorkspaceURLs []SyncerVirtualWorkspaceURL `json:"syncerVirtualWorkspaceURLs,omitempty"`
```

Comment:
Why is this a separate struct instead of []string?

Reply:
cf this previous comment from @sttts : https://github.com/kcp-dev/kcp/pull/1018#discussion_r874214619

Reply:
Thanks. I don't want to hold this up unnecessarily, but does status.syncerVirtualWorkspaceURLs[].url look/sound "good" (the repeated url)? Maybe

```
status:
  virtualWorkspaces:
    - url: https://some.url.here
    - url: https://some.url.there
```

is cleaner?

Reply:
I find it cleaner as well.

Reply:
there might more information come next to the URL. E.g. something that drives sharding.